lib/middleware/redirects.js

```diff
@@ -34,6 +34,13 @@ var Redirect = function(source, destination, type) {
 };
 
 Redirect.prototype.test = function(url) {
+  var qs = '';
```

Comment:
Should we explicitly remove the query string in all places we match (rewrites, headers etc.)?